locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -207,6 +207,8 @@
     private static final Pattern LIST_PATTERN = Pattern.compile("^\\[(.*)\\]$");
     private static final Pattern INTEGER_PATTERN = Pattern.compile("^\\d*$");
 
+    private static final int NOTES_MAX_LENGTH = 1000;
```

Comment:
It's highly unlikely that it will ever decrease as it's part of the official API description. Since it's only used for the user-metadata generated notes I don't think it's worth having it configurable (in the case the limit increases).